konekcija.py

```diff
@@ -0,0 +1,33 @@
+import psycopg2
+
+host="172.18.0.1"
+database="Data-project"
+user="postgres"
+password="Cristiano15+"
+
+def connect_to_db(host,database,user,password):
+    try:
+        connection=psycopg2.connect(
```

Comment:
Formatiranje ti je uzas, instaliraj black ili flake8 :)

Reply:
Svestan sam,sredicu to ,a svakako cu checkirati taj flake8